chroma-manager/chroma_core/migrations/0037_remove_rsyslog_types.py

```diff
@@ -0,0 +1,1537 @@
+# -*- coding: utf-8 -*-
+import datetime
+import json
+
+from functools import partial
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Q
+
+def does_not_contain_content_type_id (content_type_ids, json):
+    return json["locked_item_type_id"] not in content_type_ids
+
+def update_locks_json (content_type_ids, job):
+    locks_json = json.loads(job.locks_json)
+    locks_json = filter(partial(does_not_contain_content_type_id, content_type_ids) , locks_json)
+    job.locks_json = json.dumps(locks_json)
+    job.save()
+
+def item_in_list (items, item):
+    return item in items
+
+def update_wait_for_json_entries (job_ids, job):
+    entries = json.loads(job.wait_for_json)
+    new_entries = filter(partial(item_in_list, job_ids), entries)
+    job.wait_for_json = json.dumps(new_entries)
+    job.save()
+
+def get_id (item):
```

Comment:
toolz also has a `getter` fn.

See https://github.com/intel-hpdd/intel-manager-for-lustre/blob/master/chroma-agent/chroma_agent/device_plugins/linux_components/block_devices.py#L243 for an example.